src/main/java/com/easypost/model/ShipDateForZipPairRecommendation.java

```diff
@@ -0,0 +1,10 @@
+package com.easypost.model;
+
+import lombok.Getter;
+
+@Getter
+public class ShipDateForZipPairRecommendation {
+    private String carrier;
+    private String service;
+    private TimeInTransitDetailsForDeliveryDate easypostTimeInTransitData;
```

Comment:
You need to have a TimeInTransitDetailsForShipDate, with the following fields:

"desired_delivery_date", "ship_on_date", "delivery_date_confidence", "estimated_transit_days", "days_in_transit"

Reply:
`desired_delivery_date` should not be included, I didn't see it in the API response. ![Screenshot 2024-07-12 at 4 54 59 PM](https://github.com/user-attachments/assets/0d4fbd8b-c6d1-4a42-aa9a-b229fb74040d)